app/views/layouts/_list-of-registers.html.haml

```diff
@@ -2,6 +2,6 @@
   - @collection.registers_by_this_collection.each do |register|
     %li.list-of-registers__register
       %h3.list-of-registers__heading
-        =link_to "#{register.title} register", register_path(register.slug), {class: 'govuk-heading-s govuk-!-margin-bottom-0'}
+        =link_to "#{register.safe_name}", register_path(register.slug), {class: 'govuk-heading-s govuk-!-margin-bottom-0'}
```

Comment:
do we have a decision on when to use the SEO title and when to use the register title? As currently, this will put the SEO title in more places.

Reply:
This method was added because there were places on staging and on local that `title` wasn't being populated and `seo_title` gave a better fallback. 

I'd like to raise a ticket to streamline the names once and for all, but in the meantime I think that this method provides an acceptable stopgap.

Reply:
[Ticket raised]( https://trello.com/c/X1LeLGTf/1559-remove-the-uncertainty-around-title-seotitle-and-name) 👍